File: datasets/IR-Plag/convertai-original-java-rest-typescript/case-03/plagiarized/L2/05/inputoutput.ts
import * as readline from 'readline';

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
});

rl.question('Enter weight in pounds: ', (weightInPounds) => {
  const bobot = parseFloat(weightInPounds);

  rl.question('Enter feet: ', (feetStr) => {
    const kaki = parseFloat(feetStr);

    rl.question('Enter inches: ', (inchesStr) => {
      const inc = parseFloat(inchesStr);
      const height = kaki * 12 + inc;
      const bmi = bobot * 0.45359237 / ((height * 0.0254) * (height * 0.0254));

      console.log(`BMI is ${bmi}`);

      if (bmi < 18.5) {
        console.log('Underweight');
      } else if (bmi < 25) {
        console.log('Normal');
      } else if (bmi < 30) {
        console.log('Overweight');
      } else {
        console.log('Obese');
      }

      rl.close();
    });
  });
});

File: datasets/IR-Plag/convertai-original-java-rest-python/case-03/plagiarized/L3/04/inputoutput.py
import math

weight_lbs = float(input("Enter weight in pounds: "))
feet = float(input("Enter feet: "))
inches = float(input("Enter inches: "))

height_m = (feet * 12 + inches) * 0.0254
weight_kg = weight_lbs * 0.45359237

bmi = weight_kg / (height_m ** 2)

print(f"BMI is {bmi:.2f}")

if bmi < 18.5:
    print("Underweight")
elif bmi < 25:
    print("Normal")
elif bmi < 30:
    print("Overweight")
else:
    print("Obese")

